ios/Exponent/Versioned/Core/UniversalModules/EXFacebook/EXScopedFacebook.m

```diff
@@ -48,15 +48,16 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId andParams:(NSDicti
     BOOL hasPreviouslySetAutoInitEnabled = [_settings boolForKey:AUTO_INIT_KEY];
     BOOL manifestDefinesAutoInitEnabled = [params[@"manifest"][@"facebookAutoInitEnabled"] boolValue];
 
-    NSString *facebookAppId = params[@"manifest"][@"facebookAppId"];
+    NSString *scopedFacebookAppId = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FacebookAppID"];
+    UMLogInfo(@"Overriding Facebook App ID with the Expo Client's. To test your own Facebook App ID, you'll need to build a standalone app. Refer to our documentation for more info- https://docs.expo.io/versions/latest/sdk/facebook/");
```

Comment:
Do you think it makes sense to **always** log that information when the experience is starting? Maybe only if user defines his/her own `FacebookAppID` (if so, they may expect Expo Client to use it) would suffice?